src/parse/converters/expressions/primary/literal/readRegexpLiteral.js

```diff
@@ -0,0 +1,17 @@
+import { REGEXP_LITERAL } from 'config/types';
+
+// as far as I can tell, this only falls over if modifiers are repeated erroneously
+var regexpPattern = /^\/(?:(?=\\\/)..|[^\/])+\/[gimy]{0,4}/;
```

Comment:
Hmmm. I had no idea you didn't have to escape `/` (and apparently a whole host of other confusing things) in js regexp literals. That seems dangerous and unintuitive to me, but I'm not a native to languages with literal regexps. A quick glance at the ECMAScript (well, the first one I found, anyways: http://bclary.com/2004/11/07/#a-7.8.5) EBNF doesn't look like those are strictly legal, but they definitely work in V8. So is it better to leave regexps out entirely or introduce them with caveats?